oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/rdb/RDBDocumentNodeStoreBuilderTest.java

```diff
@@ -67,6 +68,20 @@ public void fullGCDisabled() {
         assertFalse(builder.isFullGCEnabled());
     }
 
+    @Test
```

Comment:
Same here, please consider unit testing for invalid paths.

Reply:
added in https://github.com/apache/jackrabbit-oak/pull/1551/commits/167b46c5eaa518d39bad952853ac55bed2e0b7a5